.github/workflows/test-js-client.yaml

```diff
@@ -0,0 +1,31 @@
+name: Test Raccoon JS Client
+on:
+  push:
+    paths:
+      - "clients/js/**"
+  pull_request:
+    paths:
+      - "clients/js/**"
+jobs:
+  test-js-client:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['20']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm ci
+        working-directory: clients/js
+      - name: Check formatting
+        run: npm run format
```

Comment:
Q: What happens when npm format returns with non zero exit code ? Does the the job fail there or continues to run.
If it fails directly doesn't it there will be a sequence of commit push and pipeline failures. And having three different pipeline will inform of all the failures in the checks in one go.

Reply:
yes, it fails directly

makes sense, will separate them

Reply:
done